src/main/java/pro/sky/ExceptionLess/Services/EmployeeService.java

```diff
@@ -0,0 +1,53 @@
+package pro.sky.ExceptionLess.Services;
+
+import org.springframework.stereotype.Service;
+import pro.sky.ExceptionLess.Data.Employee;
+import pro.sky.ExceptionLess.Exceptions.NoFindEmployeeException;
+import pro.sky.ExceptionLess.Interfaces.IEmployeeService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class EmployeeService implements IEmployeeService {
+    private final Map<Integer,Employee> employees;
+    private int id;
+    public EmployeeService() {
```

Comment:
```suggestion
    private int id;
    
    public EmployeeService() {
```
поля и методы лучше отделять пустой строкой